File: app/src/main/java/avanger/co/id/LupaPassword.java
package avanger.co.id;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;

public class LupaPassword extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_lupa_password);

        Button btnKembaliMain = findViewById(R.id.KembaliMain);

        btnKembaliMain.setOnClickListener((v) -> finish());
    }
}

File: app/src/main/java/avanger/co/id/LupaPassword.kt
package avanger.co.id

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import avanger.co.id.databinding.ActivityLupaPasswordBinding

class LupaPassword : AppCompatActivity() {
    private lateinit var binding: ActivityLupaPasswordBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityLupaPasswordBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.KembaliMain.setOnClickListener {
            finish()
        }
    }
}